src/ExportPDFPlugin.ts

```diff
@@ -38,15 +38,23 @@ export class ExportPDFPlugin extends Plugin {
   }
 
   initButtonCommands(): unknown {
-    return this.showButton
+    return ExportPDFPlugin.showButton
```

Comment:
This is unnecessary. If you didn't want the button, you wouldn't put it in the Licit instance.

Reply:
Removed the showButton option